contracts/mocks/MockOracle.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity 0.4.24;
```

Comment:
If possible, can you make this a move so we don't lose git history?

Reply:
Done 👍

Reply:
Most reliable way is to the rename in a separate branch then here, then rebase this one.